test/book_model_test.go

```diff
@@ -0,0 +1,127 @@
+package test
```

Comment:
hmm, why do you need separate dir for test?

Reply:
this is bad practice. In Golang your tests should be stored near implemented functionality